airgoons_mikeforce_customizations/config/arsenal.hpp

```diff
@@ -188,7 +188,7 @@ class vn_whitelisted_arsenal_loadouts
 			{"vn_m129_mag",{-1,0,0,-1}},
 			{"vn_m14_10_mag",{-1,5,0,-1}},
 			{"vn_m14_10_t_mag",{-1,5,0,-1}},
-			{"vn_m14_grenade_mag",{-1,4,0,-1}},
+			{"vn_m14_grenade_mag",{-1,-1,-1,-1}},
```

Comment:
Why is this being disabled?

Reply:
From testing in the arsenal they didn't seem to do much so at the time I figured I'd get rid of them for clutter reduction, but since then I've seen in the main discord that they actually do destroy mortars and AAs assuming you can hit with the damn things. It's fine with me if you want to revert this or leave it!